interfaces/builtin/i2c.go

```diff
@@ -73,18 +79,31 @@ func (iface *i2cInterface) String() string {
 // identification
 var i2cControlDeviceNodePattern = regexp.MustCompile("^/dev/i2c-[0-9]+$")
 
+// Pattern to match allowed i2c sysfs names.
+var i2cValidSysfsName = regexp.MustCompile("^[a-zA-Z0-9_-]+$")
```

Comment:
Is there a reference for allowed names (eg, in kernel documentation, etc)?

Reply:
I don't know if there is one, I went the conservative route.